sos/plugins/opendaylight.py

```diff
@@ -29,12 +29,12 @@ def setup(self):
         if self.get_option("all_logs"):
             self.add_copy_spec([
                 "/opt/opendaylight/data/log/",
-                "/var/log/containers/opendaylight/",
+                "/var/log/containers/opendaylight/karaf/logs/",
             ])
         else:
             self.add_copy_spec([
                 "/opt/opendaylight/data/log/*.log*",
-                "/var/log/containers/opendaylight/*.log*",
+                "/var/log/containers/opendaylight/karaf/logs/*.log*",
```

Comment:
I guess the above change and also this one is triggered by moving the logfiles of interest from `/var/log/containers/opendaylight/` to `/var/log/containers/opendaylight/karaf/logs` dir - if I am right, the change makes sense.

Reply:
Yes, exactly. Thanks.